specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-02-01/managedClusters.json

```diff
@@ -2641,6 +2753,68 @@
       },
       "description": "The time span with start and end properties."
     },
+    "RunCommandRequest": {
+      "properties": {
+        "command": {
+          "type": "string",
+          "description": "command to run."
+        },
+        "context": {
+          "type": "string",
+          "description": "base64 encoded zip file, contains files required by the command"
+        },
+        "clusterToken": {
+          "type": "string",
+          "description": "AuthToken issued for AKS AAD Server App."
+        }
+      },
+      "description": "run command request"
+    },
+    "RunCommandResult": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "command id."
+        },
+        "properties": {
+          "description": "Properties of command result.",
+          "$ref": "#/definitions/CommandResultProperties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "description": "run command result."
+    },
+    "CommandResultProperties": {
```

Comment:
>CommandResultProperties [](start = 5, length = 23)

Are the below properties read only? If so, could you mark them accordingly?

Reply:
fixed.